tests/foreman/ui/test_capsulecontent.py

```diff
@@ -234,6 +236,139 @@ def test_positive_content_counts_for_mixed_cv(
+        2. Disabled automatic content counts update.
+
+    :steps:
+        1. Create two LCEs, assign them to the Capsule.
+        2. Create and sync a repo, publish it in two CVs, promote both CVs to both LCEs.
+        3. Ensure no counts were calculated yet for both CVs, both LCEs.
+        4. Refresh counts for the first CV in the first LCE, check entity IDs in the Update task.
+        5. Ensure the counts were updated for the first CV and
+           the second CV stayed untouched, as well as the second LCE.
+        6. Refresh counts for the second LCE, check entity IDs in the Update task.
+        7. Ensure the counts were updated for the second LCE and
+           the second CV in the first LCE stayed untouched.
+
+    :expectedresults:
+        1. Content counts are updated only for the specific LCEs and CVs.
+
+    """
+    wait_query = (
+        'label = Actions::Katello::ContentView::CapsuleSync OR '
+        'Actions::Katello::CapsuleContent::UpdateContentCounts'
+    )
+
+    # 1. Create two LCEs, assign them to the Capsule.
+    lce1 = module_target_sat.api.LifecycleEnvironment(organization=function_org).create()
+    lce2 = module_target_sat.api.LifecycleEnvironment(
+        organization=function_org, prior=lce1
+    ).create()
+    module_capsule_configured.nailgun_capsule.content_add_lifecycle_environment(
+        data={'environment_id': [lce1.id, lce2.id]}
+    )
+
+    # 2. Create and sync a repo, publish it in two CVs, promote both CVs to both LCEs.
+    repo = module_target_sat.api.Repository(
+        product=function_product,
+    ).create()
+    repo.sync()
+    repo = repo.read()
+    cvs = []
+    for _ in range(2):
+        cv = module_target_sat.api.ContentView(
+            organization=function_org, repository=[repo]
+        ).create()
+        cv.publish()
+        cv = cv.read()
+        cv.version[0].promote(data={'environment_ids': [lce1.id, lce2.id]})
+        cvs.append(cv.read())
+    cv1, cv2 = cvs
+    module_target_sat.wait_for_tasks(search_query=wait_query, search_rate=5, max_tries=5)
+
+    empty_counts = [repo.name, 'N/A', 'N/A']
+    valid_counts = [
+        repo.name,
+        f'{repo.content_counts["rpm"]} Packages',
+        f'{repo.content_counts["erratum"]} Errata',
+        f'{repo.content_counts["package_group"]} Package groups',
+    ]
+
+    with module_target_sat.ui_session() as session:
+        session.capsule.edit(
+            module_capsule_configured.hostname, add_organizations=[function_org.name]
+        )
+        session.organization.select(org_name=function_org.name)
+
+        # 3. Ensure no counts were calculated yet for both CVs, both LCEs.
+        details = session.capsule.read_details(module_capsule_configured.hostname)
+        assert all(
+            details['content'][lce.name][cv.name]['expanded_repo_details'][1] == empty_counts
+            for lce in [lce1, lce2]
+            for cv in cvs
+        ), 'Expected "N/A" for content counts displayed'
```

Comment:
`not displayed`?

Reply:
I meant something like `Expected "N/A" being displayed for the content counts`.
![image](https://github.com/user-attachments/assets/3650b49f-5808-4d74-85ec-85060432c271)